client/blocks/product-selector/index.jsx

```diff
@@ -255,12 +282,14 @@ ProductSelector.propTypes = {
 			description: PropTypes.oneOfType( [ PropTypes.string, PropTypes.element ] ),
 			options: PropTypes.objectOf( PropTypes.arrayOf( PropTypes.string ) ).isRequired,
 			optionsLabel: PropTypes.string,
+			upgradeProductMap: PropTypes.object,
```

Comment:
How about defining a shape of the `upgradeProductMap` object here?

Reply:
:+1:

Reply:
👍